### ISSUE TITLE
Crash in search results

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2912]
```
Device info:
Version: 5.0.729-beta#1fe1b8d07 googleplay standard release (built 14/09/23 17:15:14)
Android version: 7.0
Android SDK version: 24
Manufacturer: HUAWEI
Model: HUAWEI VNS-L21
Storage Mb free: 3531
WebView version: 117.0.5938.61
SQLITE version: 3.42.0
Used heap memory in Mb: 15
Max heap memory in Mb: 192

Exception:
java.lang.RuntimeException: Unable to start activity ComponentInfo{net.bible.android.activity/net.bible.android.view.activity.search.SearchResults}: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.os.Parcel.dataSize()' on a null object reference
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2793)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2864)
at android.app.ActivityThread.-wrap12(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1567)
at android.os.Handler.dispatchMessage(Handler.java:105)
at android.os.Looper.loop(Looper.java:156)
at android.app.ActivityThread.main(ActivityThread.java:6517)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:832)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.os.Parcel.dataSize()' on a null object reference
at android.os.BaseBundle.(BaseBundle.java:164)
at android.os.Bundle.(Bundle.java:106)
at android.content.Intent.getExtras(Intent.java:6641)
at androidx.activity.ComponentActivity.getDefaultViewModelCreationExtras(ComponentActivity.java:655)
at androidx.lifecycle.ViewModelProviderGetKt.defaultCreationExtras(ViewModelProvider.kt:363)
at androidx.lifecycle.ViewModelProvider.(ViewModelProvider.kt:132)
at androidx.lifecycle.SavedStateHandleSupport.getSavedStateHandlesVM(SavedStateHandleSupport.kt:109)
at androidx.lifecycle.SavedStateHandlesProvider$viewModel$2.invoke(SavedStateHandleSupport.kt:134)
at androidx.lifecycle.SavedStateHandlesProvider$viewModel$2.invoke(SavedStateHandleSupport.kt:133)
at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
at androidx.lifecycle.SavedStateHandlesProvider.getViewModel(SavedStateHandleSupport.kt:133)
at androidx.lifecycle.SavedStateHandlesProvider.performRestore(SavedStateHandleSupport.kt:168)
at androidx.lifecycle.SavedStateHandleAttacher.onStateChanged(SavedStateHandleSupport.kt:199)
at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.kt:314)
at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.kt:251)
at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.kt:287)
at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.kt:136)
at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.kt:119)
at androidx.lifecycle.ReportFragment$Companion.dispatch$lifecycle_runtime_release(ReportFragment.kt:192)
at androidx.lifecycle.ReportFragment.dispatch(ReportFragment.kt:87)
at androidx.lifecycle.ReportFragment.onActivityCreated(ReportFragment.kt:50)
at android.app.Fragment.performActivityCreated(Fragment.java:2373)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1021)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1178)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1160)
at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:2046)
at android.app.FragmentController.dispatchActivityCreated(FragmentController.java:176)
at android.app.Activity.performCreateCommon(Activity.java:6908)
at android.app.Activity.performCreate(Activity.java:6918)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1123)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2746)
... 9 more
```

https://support.andbible.org/scp/tickets.php?id=2131

**Comment by tuomas2** [comment]
>  It happened only once, and after I restarted the app the specific search worked just fine. 
I searched for two words in the Afrikaans Bible, when it crashed the automated crash report came up, and I decided to mail in the hope that it may help.
Like I say, it happened once, the first time I had crash ever, and afterwards I couldn't reproduce it.

**Comment by tuomas2** [comment]
https://support.andbible.org/scp/tickets.php?id=2237

**Comment by timbze** [comment]
Duplicate of #3072
